app/models/generic_file.rb

```diff
@@ -173,16 +173,8 @@ def open_access_and_no_doi?
   end
   private :open_access_and_no_doi?
 
-  def in_collections?(collection_ids=[], collection_paths=[[{}]])
-    collection_paths.each do |collection_path|
-      collection_path.each do |collection_obj|
-        if collection_ids.include?(collection_obj[:id])
-          return true
-        end
-      end
-    end
-
-    false
+  def in_collections?(collection_array=[])
+    !(collection_array & self.collection_ids).empty?
```

Comment:
I don't think we can use this because being "in a collection" might mean being a couple layers down, e.g.:

```

   Collection A
   │
   └─Collection AA
     │
     └──Collection AAA
        │
        └──Record 1234
```

Record 1234 should have the call to `in_collections?([Collection_A])` return True which wouldn't be the case anymore here.

Reply:
Ahh, great catch! I completely overlooked that.